apps/frontend/src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -362,6 +390,25 @@ export default function Page() {
                 employees={associates}
                 setSelected={setSelectedUser}
               />
+
+              <IconButton
+                aria-label="Weekly Comments"
+                icon={<ChatIcon />}
+                onClick={openEvaluationForm}
+                minW="200px"
+              >
+                Weekly Comments
+              </IconButton>
+              {isOpenCommentForm && (
```

Comment:
It looks like we are tracking this idea of `isOpen` in both this file here, and also the `isModalOpen` state variable in the `Evaluations` file.  Do we need to track the `isOpen` boolean in both? Can we have that functionality just in the `Evaluations` file, or are they two separate things?

Reply:
can we pass down isOpenComment for the isOpen prop? Also yeah agree with izzy is it possible to drill down the props to the EvaluationModal?

Reply:
Also agree with izzy, is it possible to utilize the existing functions for opening and closing the evaluation form and pass them down to replace the isModalOpen state, handleModalOpen function, and handleModalClose function

Reply:
I tried passing it down, but had an issue making the distinction between a new comment and editing the comment. When you open the modal from pressing on an existing comment, after making changes and submitting, it was making a new comment. Will continue to look into this.